README.md

```diff
@@ -283,6 +283,28 @@ interface Strikethrough extends Parent {
 
 **Strikethrough** represents a piece of text that has been stricken.
 
+### `Subscript`
```

Comment:
**todo:** If I'm not mistaken the [json schemas](https://github.com/Financial-Times/content-tree/tree/main/schemas) will need to be updated.
For context, the C&M team will allow publishing of content tree next year and those schemas will be our source of truth for validation.

Reply:
The JSON schemas get automatically generated as part of the [build step](https://github.com/Financial-Times/content-tree/blob/main/build.bash)

I _think_ what's missing (and why the JSON schemas haven't been updated yet) is that the new types need to be included as things that can be "Phrasing" - https://github.com/Financial-Times/content-tree/pull/57/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R130

Reply:
I've just added `Subscript` and `Superscript` to the `Phrasing` types, and re-built the schema 👍